app/src/main/java/com/xmartlabs/template/common/exeption/EntityNotFoundException.java

```diff
@@ -1,7 +1,7 @@
 package com.xmartlabs.template.common.exeption;
 
 /**
- * Created by mirland on 18/01/17.
+ * To be thrown if an entity cannot be recovered from the database
```

Comment:
I see it's inline here in some places. I don't know the rest, but I prefer it that way.